octopus_deploy/tests/fixtures/GET/api/Spaces-1/machines/skip=0/take=2/response.json

```diff
@@ -0,0 +1,68 @@
+{
+    "ItemType": "Machine",
+    "TotalResults": 1,
+    "ItemsPerPage": 2,
+    "NumberOfPages": 1,
+    "LastPageNumber": 0,
+    "Items": [
+        {
+            "Id": "Machines-1",
+            "EnvironmentIds": [
+                "Environments-1"
+            ],
+            "Roles": [
+                "test-tag"
+            ],
+            "TenantedDeploymentParticipation": "Untenanted",
+            "TenantIds": [],
+            "TenantTags": [],
+            "SpaceId": "Spaces-1",
+            "Name": "test-machine",
+            "Thumbprint": "test-footprint",
+            "Uri": "hostname-test:10933/",
+            "IsDisabled": false,
+            "MachinePolicyId": "MachinePolicies-1",
+            "HealthStatus": "Healthy",
```

Comment:
We're only testing `HealthStatus` with `Healthy` value

Reply:
Added multiple health status values!